From e3f406950268fe6b4004548135023699985e131b Mon Sep 17 00:00:00 2001
From: xingxing <xingxing@raysengine.com>
Date: Sun, 13 Aug 2023 18:43:18 +0800
Subject: [PATCH] Remove `-fexcess-precision` since it's not available util
 clang 16

I'm locked by Arch, where llvm 16 is still not available...
---
 arch/x86/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/x86/Makefile b/arch/x86/Makefile
index 3828a4745..63ca6c4e7 100644
--- a/arch/x86/Makefile
+++ b/arch/x86/Makefile
@@ -62,7 +62,7 @@ export BITS
 
 # More compiler optimizations
 ifdef CONFIG_CC_OPTIMIZE_FOR_PERFORMANCE
-KBUILD_CFLAGS += -O3 -fexcess-precision=fast -freciprocal-math \
+KBUILD_CFLAGS += -O3 -freciprocal-math \
 		 $(call cc-option,-fcx-limited-range)
 KBUILD_RUSTFLAGS += -Copt-level=3
 endif
-- 
2.41.0

